packages/fuel-indexer-tests/components/indices/fuel-indexer-test/fuel_indexer_test.yaml

```diff
@@ -2,7 +2,7 @@ namespace: fuel_indexer_test
 graphql_schema: packages/fuel-indexer-tests/components/indices/fuel-indexer-test/schema/fuel_indexer_test.graphql
 abi: packages/fuel-indexer-tests/contracts/fuel-indexer-test/out/debug/fuel-indexer-test-abi.json
 start_block: 1
-contract_id: fuel1dthcclayuwhce7t7jt8dlpvv2c0q4wsyj4xsudcek5gwc6nd5cnqj4qkua
+contract_id:
```

Comment:
in other places where we have a null field in yaml I've seen `~`, do we want that here?

Reply:
I can try it, but I think it may need to be empty in order for `#contract_conditional` in `indexer.rs` to not try to parse it into a contract ID. With it being empty, the WASM module will listen to all contracts anyways.

@ra0x3, thoughts?

Reply:
@deekerno 
- `~` is `null` in YAML so the `if let Some(contract_id)` part of the codegen should never be hit.
- If we can let's try to use explicit null `~` instead of implied null (blank space)

Reply:
@ra0x3 Using the explicit null worked.